Marlin/src/pins/esp32/pins_MKS_TINYBEE.h

```diff
@@ -112,8 +112,13 @@
 #define FAN1_PIN                             148  // FAN2
 #define HEATER_BED_PIN                       144  // H-BED
 
-//#define CONTROLLER_FAN_PIN                 148  // FAN2
-//#define E0_AUTO_FAN_PIN                    148  // FAN2
+#ifndef E0_AUTO_FAN_PIN
+  #define E0_AUTO_FAN_PIN               FAN1_PIN  // FAN2
```

Comment:
I've been conscientiously defining more pins using the pin number rather than references to other defined pins, especially those that have an optional function w/r/t Marlin. This is in consideration of the possibility of some configuration variant that includes `#undef FAN1_PIN` (because why define a pin that's not needed?) which would then break the definition of `E0_AUTO_FAN_PIN`. Although in most cases fan pins do correspond well to the pins marked for fans on the board, that is not strictly the way fan pins _must_ be defined.

Reply:
Using `FAN1_PIN` for `E0_AUTO_FAN_PIN`, `FAN2_PIN` for `E1_AUTO_FAN_PIN`, etc. is a common configuration and has been around since as long as I have been using Marlin (since at least 2018), so it would be *really* tough to see that go away.

Reply:
Defining pins in reference to other pins is fine when they are known to be unaltered and works best in the case of a configuration file. But pins files are only meant to define the best defaults for any board, while also leaving open the potential for overrides and reordering. Tying the default of one pin to the value of another, which might be modified, is bound to cause confusion, so I am inclined to avoid this inside of pins files going forward. A common internal default used only within a pins file is another matter, as individual pins files are free to define pins according to their AUX number, silkscreen name, etc.